Added the not yet public collier code to the openloops repository (As…

thuer opened the pull request:
Added the not yet public collier code to the openloops repository (As authorized by the authors)

thuer committed:
Added the not yet public collier code to the openloops repository (As authorized by the authors)

cmsbuild commented:
A new Pull Request was created by  @thuer (Sebastian Thüer) for branch 1.2.4.

@cmsbuild, @smuzaffar, @degano, @iahmad-khan, @davidlange6 can you please review it and eventually sign? Thanks.
You can sign-off by replying to this message having '+1' in the first line of your reply.
You can reject by replying  to this message having '-1' in the first line of your reply.

external issue cms-sw/cmsdist#2134

ghost commented:
@thuer It is not clear to me the purpose of this PR: basically we would never use this added code since the branch cms/v1.2.4 wouldn't have it. Perhaps it would be better to include everything in the cms branch? Also because this branch should reflect what there actually is inside the vanilla 1.2.4.

thuer commented:
@Degano I added the same commit to the cms/v1.2.4 branch. The reason for this code is that besides the public 1.2.4 release of OpenLoops there is a private one (in sync with 1.2.4)  which adds the collier libraries(they are not officially public yet). I added these extra code to both branches since it is the one we would like to use as baseline

ghost commented:
@thuer Okay, thanks for the clarification. I'll merge this and test the CMS one.
